src/lib/packager/clients/pkg_finish.rb

```diff
@@ -288,8 +289,15 @@ def sync_target_sources
     #
     # @see Yast::Packager::CFA::ZyppConf#set_minimalistic!
     def set_minimalistic_libzypp_conf
```

Comment:
As it is now more generic I'd rename the method. It's safe, it's a private method.

```suggestion
    def set_minimalistic_pkg_conf
```